fragments/k8s/cdk/bundle.yaml

```diff
@@ -5,7 +5,7 @@ series: xenial
 services:
   "kubernetes-master":
     charm: "cs:~containers/kubernetes-master"
-    num_units: 1
+    num_units: 2
```

Comment:
@CalvinHartwell Is this what CPE does for customers by default (2 masters)?

Reply:
Yes, we need failover / ha of master